config/initializers/sidekiq.rb

```diff
@@ -0,0 +1,9 @@
+Sidekiq.configure_server do |config|
+  # Calls to Rails.logger in a sidekiq process will use Sidekiq's logger
+  Rails.logger = Sidekiq::Logging.logger
```

Comment:
I assume this bit of config was copied from another repo. Could you link to https://github.com/alphagov/email-alert-api/commit/529711e8d72a272c352a373951a348f179c28d4b from your commit message? That way we have an easy way of finding out why the config was added.